profiles/migrations/0008_profile_edrpou_char.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.3 on 2024-02-06 18:35
+
+from django.db import migrations, models
+import validation.validate_edrpou
+
+def convert_edrpou_int_filed_to_char(app, schema_editor):
+    Profile = app.get_model("profiles", "Profile")
+    for instance in Profile.objects.all():
+        instance.edrpou_char = str(instance.edrpou) if instance.edrpou is not None else None
+        instance.save()
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("profiles", "0007_profile_completeness"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="profile",
+            name="edrpou_char",
+            field=models.CharField(
+                max_length=8,
+                default=None,
+                null=True,
+                unique=True,
+                validators=[validation.validate_edrpou.validate_edrpou],
+            ),
+        ),
+        migrations.RunPython(convert_edrpou_int_filed_to_char, migrations.RunPython.noop),
```

Comment:
Probably this migration should be elidable?

Reply:
Done